README.md

```diff
@@ -36,13 +36,13 @@ $ npm install
 
 Create a `.env` file inside the `/backend` directory (`touch .env` in your terminal or manually create file using your IDE)
 
-Add the following content into the `.env` file:
+Add the following content into the `.env` file (You will have the required credentials if you area authorized to continue development on this project):
```

Comment:
Is there a way we could write this so that anyone knew the accepted types of values so they knew how they could swap in their own? E.g., 

Add the following content into the `.env` file (entering your own corresponding Mongoose and JWT values)
DB_USER=[mongoose db user]
DB_PASSWORD=[mongoose db password]
DB_NAME=[mongoose db name]
JWT_SECRET=[JSON web token secret]

(maybe re-word a bit if those aren't the best descriptors)